LayoutTests/imported/w3c/web-platform-tests/css/css-values/lh-unit-same-element-line-height-dependency-expected.xht

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```

Comment:
Is there any reason for this to be XHTML?

Reply:
The reference file (`<link rel=match>`) this is referring to is XHTML, so if this were to be imported using the `import-w3c-tests` script, it would be copied as a XHTML file.